tests/system/src/test/java/test/robot/javafx/scene/MouseLocationOnScreenTest.java

```diff
@@ -43,7 +43,7 @@
 public class MouseLocationOnScreenTest {
     static CountDownLatch startupLatch = new CountDownLatch(1);
     static Robot robot;
-    private static int DELAY_TIME = 1;
+    private static int DELAY_TIME = 3;
```

Comment:
minor: `Util.sleep()` could be moved to `validate()`

question: even though the value is 3x the original, should it be longer to account for any hiccups?  maybe 10?  what do you think?

Reply:
If we go with this solution, I would recommend leaving it at 3ms. It fails very, very rarely with the existing 1ms, so 3x the time ought to be enough. Tripling it again would just make the test take that much longer without any real assurance that it is needed.